src/md/powered-by-apex.md

```diff
@@ -5,7 +5,7 @@ This page lists organizations and software projects which work with Apache Apex.
 
 ## Companies
 
-Apache Apex powers
```

Comment:
http://www.apache.org/foundation/marks/guide
Using Apache Hadoop vs Hadoop section
provides guidance on using the Apache brand name. 

It talks about mentioning Apache in the most prominent part of the page like title and headers which we cover at the top of the page. 

As an aside, I have seen no consistent style in other pages it is a mixture of Apache Apex and Apex. We should address those as well.